my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
```

Comment:
for the way you are using matplotlib in the rest of your script this import should be 
```python
from matplotlib import pyplot
```
however, it is more common to import pyplot like this:
```python
import matplotlib.pyplot as plt
```
which means you will need to replace `pyplot` with `plt` everywhere else.